harness/determined/pytorch/lightning/_adapter.py

```diff
@@ -120,7 +121,14 @@ class LightningAdapter(PyTorchTrial):
     simple job submission to the Determined cluster, and so on.
     """
 
-    def __init__(self, context: PyTorchTrialContext, lightning_module: pl.LightningModule):
+    def __init__(
+        self,
+        context: PyTorchTrialContext,
+        lightning_module: pl.LightningModule,
+        precision: Union[Literal[32], Literal[16]] = 32,
+        amp_backend: Union[Literal["native"], Literal["apex"]] = "native",
+        amp_level: Union[Literal["O0", "O1", "O2", "O3"]] = "O2",
```

Comment:
Should this be `O0`?

Reply:
it's visually confusing but it is a capital O + the number 0 from what I can tell. I confirmed by searching for the expected value.